install.go

```diff
@@ -142,7 +137,6 @@ var (
 		kcoordinationv1beta1.AddToScheme,
 		keventsv1.AddToScheme,
 		keventsv1beta1.AddToScheme,
-		kextensionsv1beta1.AddToScheme,
```

Comment:
same problem with scale.  This will prevent the ability to read such input from disk for instance.  If you demonstrate workingness is all our client using it (oc and openshift-apiserver are probably the most notable), then this coudl be merged, but not before due to risk of wedging.